### DISCUSSION TITLE
Why was coc-bookmark deprecated?

### DISCUSSION THREAD

**Discussion by thedracle** [neoclide/coc.nvim#3119] [Extension request]
I'm very keen on the functionality in coc-bookmark. It helps me make sense of large projects. I used https://vimawesome.com/plugin/vim-bookmarks for some time, but had various issues with it.

Is there a reason coc-bookmark was deprecated? Could I fork it perhaps?

**Comment by fannheyward** [comment]
coc-bookmark was developed by @voldikss. Ping @voldikss for more informations.

**Comment by voldikss** [comment]
Hi @thedracle , coc-bookmark has a terrible implementation, dispointedly I do not have any time to improve it anymore. 

Also I do not want others to take a look into the code because it doesn't make any sense. If someone wants to write a new one, it's better not to be affected by the bad implementation of coc-bookmark. So sorry for any conveniences caused by the deprecation.
